lib/atspi/atspi_node.cc

```diff
@@ -332,15 +339,16 @@ AtspiNode AtspiNode::getChildAtIndex(int index) const {
 std::vector<AtspiNode> AtspiNode::getChildren() const {
   std::vector<AtspiNode> result;
 
-  GError* error = nullptr;
-  gint child_count = atspi_accessible_get_child_count(accessible_, &error);
-  if (error) {
+  ScopedGError error;
+  GError* error_ptr = error.get();
+  gint child_count =
+      atspi_accessible_get_child_count(accessible_.get(), &error_ptr);
+  if (error_ptr) {
     std::string err_msg = error->message;
-    g_error_free(error);
     throw std::runtime_error(err_msg);
   }
 
-  result.resize(child_count);
+  result.reserve(child_count);
```

Comment:
Hmmm good catch! I hadn't spotted that we were using `result[i]` below, meaning that `resize()` actually _is_ sensible here, because of the `child_node.isNull()` check on line 355 if nothing else (i.e. we'd end up with undefined members of the vector, yikes). And yeah, looks like it's undefined behaviour to assign those nonexistent elements too.

I'll just revert this change. `resize()` bugs me in general because it default initializes all the members, but in this case it seems like it's the right thing to use!